applications/GeoMechanicsApplication/custom_operations/activate_model_part_operation.h

```diff
@@ -56,7 +56,7 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) ActivateModelPartOperation : public
     ///@{
 
     /// Default constructor.
-    ActivateModelPartOperation() : Operation() {}
+    ActivateModelPartOperation();
```

Comment:
I think it'd be cleaner to just have =default in the header here:

```suggestion
    ActivateModelPartOperation() = default
```

Reply:
done